evently/android/app/build.gradle

```diff
@@ -52,7 +52,7 @@ android {
     }
 
     signingConfigs {
-        debug {
+        release {
```

Comment:
could we try to add both stages here? and see? same config, just copy paste.

Reply:
I tried but it shows
 A problem occurred evaluating project ':app'